package.json

```diff
@@ -1,36 +1,38 @@
 {
-  "name": "lock",
-  "version": "1.7.0",
+  "name": "auth0-custom-lock",
+  "version": "2.0.0",
   "description": "The front-end for Mozilla's Auth0 Lock",
   "main": "index.js",
   "scripts": {
-    "start": "gulp dev",
-    "build": "gulp build"
+    "build": "gulp build",
+    "deploy": "a0deploy import -f yaml -i config/a0config.yml",
+    "watch": "NODE_ENV=local gulp dev"
   },
-  "author": "Hidde de Vries",
+  "author": "April King",
```

Comment:
I mean Hidde did create NLX. This was his work over a couple years.

Reply:
Fixed.